plugins/analysis/java-plugin/src/main/kotlin/arrow/meta/plugins/analysis/java/ast/descriptors/JavaFieldDescriptor.kt

```diff
@@ -29,7 +29,7 @@ public class JavaFieldDescriptor(
     require(impl.kind == ElementKind.FIELD)
   }
 
-  override val isVar: Boolean = true
+  override val isVar: Boolean = !impl.modifiers.contains(Modifier.FINAL)
```

Comment:
Does Javac has anything related to the support in Java15 for Records? This may not be true in Records AFAIK.